akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/Effect.scala

```diff
@@ -25,18 +25,21 @@ import scala.concurrent.duration.FiniteDuration
 abstract class Effect private[akka] ()
 
 object Effect {
+
```

Comment:
Yes, a space from type def before inner code is good. I see a lot w/out a space.